server/java/src/main/java/com/godaddy/asherah/grpc/ServerApp.java

```diff
@@ -82,7 +116,15 @@ public Void call() throws Exception {
 
     KeyManagementService keyManagementService =
         appEncryptionConfig.setupKeyManagementService(kmsType, preferredRegion, regionMap);
-    Metastore<JSONObject> metastore = appEncryptionConfig.setupMetastore(metastoreType, jdbcUrl);
+
+    DynamoDbConfig dynamoDbConfig = new DynamoDbConfig(DynamoDbFlags.EndPoint.endpoint,
+        DynamoDbFlags.EndPoint.signingRegion,
+        DynamoDbFlags.Region.region,
+        keySuffix,
+        dynamoDbTableName);
```

Comment:
```suggestion
    DynamoDbConfig dynamoDbConfig = new DynamoDbConfig(DynamoDbFlags.EndPoint.endpoint,
        DynamoDbFlags.EndPoint.signingRegion, DynamoDbFlags.Region.region, keySuffix, dynamoDbTableName);
```

Reply:
the requirement of having each argument on a new line is not required by java checkstyle plugin